CHANGELOG.md

```diff
@@ -65,6 +65,7 @@
 - PR #4101 Redux serialize `Buffer` directly with `__cuda_array_interface__`
 - PR #4098 Remove legacy calls from libcudf strings column code
 - PR #4111 Use `Buffer`'s to serialize `StringColumn`
+- PR #4133 Mask cleanup and fixes: use `int32` dtype, ensure 64 byte padding, handle offsets
```

Comment:
I see only the changes of moving to enum class. Where is the 64-byte padding and offset stuff, is that only in Python?